src/js/components/Ballot/PositionRowListCompressed.jsx

```diff
@@ -387,13 +473,14 @@ class PositionRowListCompressed extends Component {
     );
   }
 }
+
 PositionRowListCompressed.propTypes = {
   ballotItemWeVoteId: PropTypes.string.isRequired,
+  firstInstance: PropTypes.bool,
+  positionListFromFriendsHasBeenRetrieved: PropTypes.array,
```

Comment:
I see you've added this new prop, but I don't see anywhere in the code where you pass a variable into PositionRowListCompressed. That is ok, but I just wanted to highlight this.